src/constants/LayoutConstants.js

```diff
@@ -0,0 +1,114 @@
+export const CARD_SIZES = {
+  XSMALL: 'XSMALL',
+  SMALL: 'SMALL',
+  TALL: 'TALL',
+  MEDIUM: 'MEDIUM',
+  WIDE: 'WIDE',
+  LARGE: 'LARGE',
+  XLARGE: 'XLARGE',
+};
+
+export const CARD_TYPES = {
+  LINE: 'LINE',
+  VALUE: 'VALUE',
+  TABLE: 'TABLE',
+};
+
+export const DASHBOARD_SIZES = {
+  XLARGE: 'xl',
+  LARGE: 'lg',
+  MEDIUM: 'md',
+  SMALL: 'sm',
+  XSMALL: 'xs',
+};
+
+export const DASHBOARD_COLUMNS = {
+  xl: 12,
```

Comment:
Is this "grid" supposed to emulate the Carbon 10 grid design?  If so, they are using a 16 column grid.

https://www.carbondesignsystem.com/guidelines/layout

Reply:
yeah I noticed that too @stuckless,  @bryancboyd I'm surprised we ended up with 12 columns instead of 16 total?